### DISCUSSION TITLE
https://seul1230.github.io/coding/sseul-boj-17069-move-pipe-2

### DISCUSSION THREAD

**Discussion by giscus** [seul1230/giscus#7] [Comments]
# https://seul1230.github.io/coding/sseul-boj-17069-move-pipe-2

sseul's blog

https://seul1230.github.io/coding/sseul-boj-17069-move-pipe-2

**Comment by DSL2e** [comment]
와 너무 똑똑하신거 같아요~ 어떻게 이런 생각을 ... 한 수 배우고 갑시다
